packages/hd/src/keychain.ts

```diff
@@ -1,6 +1,7 @@
-import crypto from "crypto";
-import { ec as EC } from "elliptic";
+/* eslint-disable @typescript-eslint/no-magic-numbers */
```

Comment:
`keychain.ts` has 7 magic numbers so this disable `no-magic-numbers` rule for the whole file 
just like `mnemonic/index.ts`.

If disable the rule line by line, it will break `codecov/patch`